### PR TITLE
46 file updates for various VU#s

### PR DESCRIPTION
**Pull request by madcatter24**

### CODE REVIEW

File: 2016/6xxx/CVE-2016-6564.json [hunk 1]
```diff
@@ -1,18 +1,82 @@
 {
```
Comment (cve-team) [inline 2016/6xxx/CVE-2016-6564.json]: u'version_value' is a required property

File: 2016/9xxx/CVE-2016-9496.json [hunk 2]
```diff
@@ -1,18 +1,91 @@
 {
```
Comment (cve-team) [inline 2016/9xxx/CVE-2016-9496.json]: u'version_value' is a required property
u'version_value' is a required property
u'version_value' is a required property

File: 2016/6xxx/CVE-2016-6567.json [hunk 3]
```diff
@@ -1,18 +1,76 @@
 {
```
Comment (cve-team) [inline 2016/6xxx/CVE-2016-6567.json]: u'version_value' is a required property
u'version_value' is a required property

File: 2016/9xxx/CVE-2016-9485.json [hunk 4]
```diff
@@ -1,18 +1,72 @@
 {
```
Comment (cve-team) [inline 2016/9xxx/CVE-2016-9485.json]: u'version_value' is a required property

File: 2016/6xxx/CVE-2016-6545.json [hunk 5]
```diff
@@ -1,18 +1,77 @@
 {
```
Comment (cve-team) [inline 2016/6xxx/CVE-2016-6545.json]: u'version_value' is a required property

File: 2016/6xxx/CVE-2016-6547.json [hunk 6]
```diff
@@ -1,18 +1,82 @@
 {
```
Comment (cve-team) [inline 2016/6xxx/CVE-2016-6547.json]: u'version_value' is a required property

File: 2016/9xxx/CVE-2016-9497.json [hunk 7]
```diff
@@ -1,18 +1,91 @@
 {
```
Comment (cve-team) [inline 2016/9xxx/CVE-2016-9497.json]: u'version_value' is a required property
u'version_value' is a required property
u'version_value' is a required property

File: 2016/6xxx/CVE-2016-6543.json [hunk 8]
```diff
@@ -1,18 +1,77 @@
 {
```
Comment (cve-team) [inline 2016/6xxx/CVE-2016-6543.json]: u'version_value' is a required property

File: 2016/6xxx/CVE-2016-6542.json [hunk 9]
```diff
@@ -1,18 +1,77 @@
 {
```
Comment (cve-team) [inline 2016/6xxx/CVE-2016-6542.json]: u'version_value' is a required property

File: 2016/6xxx/CVE-2016-6563.json [hunk 10]
```diff
@@ -1,18 +1,151 @@
 {
```
Comment (cve-team) [inline 2016/6xxx/CVE-2016-6563.json]: u'version_value' is a required property
u'version_value' is a required property
u'version_value' is a required property
u'version_value' is a required property
u'version_value' is a required property
u'version_value' is a required property
u'version_value' is a required property
u'version_value' is a required property
u'version_value' is a required property

File: 2016/6xxx/CVE-2016-6546.json [hunk 11]
```diff
@@ -1,18 +1,77 @@
 {
```
Comment (cve-team) [inline 2016/6xxx/CVE-2016-6546.json]: u'version_value' is a required property

File: 2016/9xxx/CVE-2016-9495.json [hunk 12]
```diff
@@ -1,18 +1,91 @@
 {
```
Comment (cve-team) [inline 2016/9xxx/CVE-2016-9495.json]: u'version_value' is a required property
u'version_value' is a required property
u'version_value' is a required property

File: 2016/9xxx/CVE-2016-9486.json [hunk 13]
```diff
@@ -1,18 +1,72 @@
 {
```
Comment (cve-team) [inline 2016/9xxx/CVE-2016-9486.json]: u'version_value' is a required property

File: 2016/6xxx/CVE-2016-6548.json [hunk 14]
```diff
@@ -1,18 +1,82 @@
 {
```
Comment (cve-team) [inline 2016/6xxx/CVE-2016-6548.json]: u'version_value' is a required property

File: 2016/6xxx/CVE-2016-6544.json [hunk 15]
```diff
@@ -1,18 +1,77 @@
 {
```
Comment (cve-team) [inline 2016/6xxx/CVE-2016-6544.json]: u'version_value' is a required property

File: 2016/6xxx/CVE-2016-6549.json [hunk 16]
```diff
@@ -1,18 +1,82 @@
 {
```
Comment (cve-team) [inline 2016/6xxx/CVE-2016-6549.json]: u'version_value' is a required property

File: 2016/9xxx/CVE-2016-9494.json [hunk 17]
```diff
@@ -1,18 +1,91 @@
 {
```
Comment (cve-team) [inline 2016/9xxx/CVE-2016-9494.json]: u'version_value' is a required property
u'version_value' is a required property
u'version_value' is a required property

File: 2016/6xxx/CVE-2016-6558.json [hunk 18]
```diff
@@ -1,18 +1,73 @@
 {
```
Comment (cve-team) [inline 2016/6xxx/CVE-2016-6558.json]: CVE id in JSON (CVE-2016-6557) does not agree with filename (CVE-2016-6558)!

**Comment by madcatter24** [comment]
When there is no version listed (because it is a mobile application or software without an affected version number available) what is the correct way to complete "affected",  "version_name", and "version_value"?

**Comment by cve-team** [comment]
Validation requires the version_value attribute to be present but doesn't otherwise check it beyond that it's a string. So for cases like these you could use a value such as "n/a".

File: 2016/6xxx/CVE-2016-6543.json [hunk 19]
```diff
@@ -1,18 +1,77 @@
...
+      }
+   ],
+   "data_format": "MITRE",
+   "data_type": "CVE",
+   "data_version": "4.0",
+   "description": {
+      "description_data": [
+         {
+            "lang": "eng",
+            "value": "A captured MAC/device ID can be registered under multiple user accounts allowing access to getgps GPS data, which can allow unauthenticated parties to track the device."
```
Comment (cve-team) [inline 2016/6xxx/CVE-2016-6543.json]: The product name and version (when that makes sense) should appear in the description as well as in the "affects" container.

File: 2016/9xxx/CVE-2016-9494.json (hunk 17 above)
Comment (cve-team) [inline 2016/9xxx/CVE-2016-9494.json]: u'version_value' is a required property
u'version_value' is a required property
u'version_value' is a required property

Comment (madcatter24) [reply 2016/9xxx/CVE-2016-9494.json]: Does this require any additional changes?

**Comment by madcatter24** [comment]
What additional changes need to be made? I've since updated my Github/git to fix the "no_user"/signature warning issue but I don't believe there's a way to adjust it in the pull request.

**Comment by cve-team** [comment]
There are a few entries with descriptions that do not have the product and/or version info (when that makes sense) . CVE-2016-6543, CVE-2016-9482, and CVE-2016-9483 are ones I noticed, and there may be others.

**Pull request merged**